release_notes/373-release-notes.adoc

```diff
@@ -15,6 +15,12 @@ toc::[]
 |`3.73.0` |6 December 2022
 |====
 
+[WARNING]
+====
+Due to a critical bug discovered in {product-title-short} version 3.73.0, we recommend that you _do not_ upgrade to this version. For more information, review the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/6990153[After upgrading the RHACS to version 3.73, the Central pod takes several hours to move into the READY state]. We are currently working to address this issue. If you have upgraded to 3.73.0 and encounter the issue, please contact Red Hat support.
```

Comment:
I'd remove the recommend and just say do not.

I also don't know if we should say "we're working on this issue". I'd recommend removing it unless people really think it's necessary. I also think we normally want to avoid "we" when possible, per the ISG

Also a few other nits

```suggestion
Due to a critical bug discovered in {product-title-short} version 3.73.0, _do not_ upgrade to this version. For more information, review the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/6990153[After upgrading the RHACS to version 3.73, the Central pod takes several hours to move into the READY state]. If you have upgraded to 3.73.0 and encounter the issue, contact Red Hat Support.
```

Reply:
Got OK from PM to make these changes. TY!